pkg/server/policy/validate.go

```diff
@@ -0,0 +1,144 @@
+package policy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+// validatePolicy runs a few sample inputs with the policy just to make sure/
+// it doesn't throw any errors
+func (e *Engine) validatePolicy() error {
```

Comment:
This should have the context threaded through from NewEngineFromRego.

Reply:
Ah! One slipped through the cracks. Fixed!